audio/filter/af_lavcac3enc.c

```diff
@@ -195,9 +195,6 @@ static void process(struct mp_filter *f)
         case MP_FRAME_AUDIO:
             TA_FREEP(&s->in_frame);
             s->in_frame = input.data;
-            frame = mp_frame_to_av(input, NULL);
-            if (!frame)
-                goto error;
```

Comment:
There was one benefit of doing it before, because even though the frame was not used, `mp_frame_to_av` do some validation when creating, so it would error. Now it would just pass through.

Although I don't see why this would matter in practice. I guess it is fine.

Alternative would be to free the frame, and keep old code flow... but I prefer current patch too.